vulcanai2/models/cnn.py

```diff
@@ -271,20 +273,22 @@ def _build_conv_network(self, conv_hid_layers, activation):
 
         """
         conv_hid_layers[0]['in_channels'] = self.in_dim[0]
-        conv_layers = []
-        for conv_layer_config in conv_hid_layers:
+        conv_layers = OrderedDict()
+        for idx, conv_layer_config in enumerate(conv_hid_layers):
             conv_layer_config['activation'] = activation
-            conv_layers.append(ConvUnit(**conv_layer_config))
-        conv_network = nn.Sequential(*conv_layers)
+            layer_name = 'conv_{}'.format(idx)
```

Comment:
Can we generate a random number as ID for the network, say `network_ID`, and link to the layer names? I know it will increase the verbosity level but I feel it will be useful for complex multi-input networks. Thoughts?

Reply:
well we already have the network name as a unique Identifier so we won't need to generate another ID. I will try to add it to when we assign the input_networks so as to tie the input names to the names of the input networks. 
<img width="567" alt="screen shot 2018-11-02 at 3 43 38 pm" src="https://user-images.githubusercontent.com/12278348/47937151-3803fe00-deb6-11e8-8fc9-36eff38e26d6.png">

For instance here, instead of `0` it can say 'Conv_1D_name'. What do you say @priyatharsan ?

Reply:
Yeah I agree on that. I was actually suggesting to put more info like network_ID_layer_name (e.g.: `net1234_conv1D_1`, `net1234_flatten`, `net1234_classifier`). I'm good with just the layer_name also.

Reply:
the name would be a little too repetitive having `net1234` at each layer. if we can attach it at the  ConvNet level that would be the cleanest implementation?

Reply:
Yeah I thought the same at first. But, since `nn.ModuleList` is more like a list not sure if it will support specifying names similar to `nn.Sequential`, however, it looks like there is a way: https://github.com/pytorch/pytorch/issues/3499